### PR TITLE
update documentation

### LINKED ISSUE #457: email-service: Write user guide

### LINKED ISSUE #482: Check if documentation is up to date
Since there were some new features added, the documentation may be out of date. Please check and update where necessary.

### PR DESCRIPTION
**Pull request by daniel-arnauer**
Document new features that have not been added to documentation yet

- [x] #438  History search (not merged yet)
- [x] #425  Searchbar for subproejects

Extra issue for documentation for email notification: #457
Closes #482

### CODE REVIEW

File: doc/wiki/User-Guide/Projects/Project.md [hunk 1]
```diff
@@ -222,6 +224,22 @@ The history contains all activities done directly refer to the current project.
 
 ![show project history](../../uploads/Screenshots/view_project_history.jpg)
 
+## Filter the history of a project
+
+**Description:**
+
+The entries in the history can be filtered by 4 different values: publisher, type of event, a start date and a end date, where a timeframe can be determined. When clicking o the search button, all values that are specified are used in the filter request.
```
Comment (Stezido) [inline doc/wiki/User-Guide/Projects/Project.md:231]: When clicking o the search button...
Suggestion:
By clicking the search button...

File: doc/wiki/User-Guide/Projects/Subproject.md [hunk 2]
```diff
@@ -52,6 +54,30 @@ View all subprojects where the current user has view-permissions on.
 
 4. At the bottom half of the site all subprojects of the current project are viewed
 
+## Filter subprojects
+
+**Description:**
+
+There are 3 ways how projects can be filtered on the overview page:
```
Comment (Stezido) [inline doc/wiki/User-Guide/Projects/Subproject.md:61]: 3 or 2?

File: doc/wiki/User-Guide/Projects/Subproject.md [hunk 3]
```diff
@@ -52,6 +54,30 @@ View all subprojects where the current user has view-permissions on.
...
+
+1. Searchbar
+2. URL
+
+The Searchbar can be used to search all projects for a term included in name, tag or status. Prefixes can be used to specify the search context (e.g. tag:mycustomtag). If no prefix is used display name, tag and status is searched for a match. After typing the project list and the URL are instantly updated. The URL can then be shared to other users including the filter.
```
Comment (Stezido) [inline doc/wiki/User-Guide/Projects/Subproject.md:66]: Prefixes can be used.... -> These can also be used as prefixes to ...
is searched -> are searched
instantly updated -> updated instantly

File: doc/wiki/User-Guide/Projects/Subproject.md [hunk 4]
```diff
@@ -197,6 +223,23 @@ The history contains all activities done directly refer to the current subprojec
 
 ![show subproject history](../../uploads/Screenshots/view_subproject_history.jpg)
 
+## Filter the history of a subproject
+
+**Description:**
+
+The entries in the history can be filtered by 4 different values: publisher, type of event, a start date and a end date, where a timeframe can be determined. When clicking o the search button, all values that are specified are used in the filter request.
```
Comment (Stezido) [inline doc/wiki/User-Guide/Projects/Subproject.md:230]: When clicking o the search button -> by clicking the search button

File: doc/wiki/User-Guide/Projects/Project.md (hunk 1 above)
Comment (daniel-arnauer) [reply doc/wiki/User-Guide/Projects/Project.md:231]: done

Comment (daniel-arnauer) [reply doc/wiki/User-Guide/Projects/Project.md:231]: 2 - done

File: doc/wiki/User-Guide/Projects/Subproject.md (hunk 2 above)
Comment (daniel-arnauer) [reply doc/wiki/User-Guide/Projects/Subproject.md:61]: 2 - done

File: doc/wiki/User-Guide/Projects/Subproject.md (hunk 3 above)
Comment (daniel-arnauer) [reply doc/wiki/User-Guide/Projects/Subproject.md:66]: done - also done for Projects.md

File: doc/wiki/User-Guide/Projects/Subproject.md (hunk 4 above)
Comment (daniel-arnauer) [reply doc/wiki/User-Guide/Projects/Subproject.md:230]: done

**Pull request merged**